stl/inc/cmath

```diff
@@ -1330,6 +1340,36 @@ _NODISCARD constexpr _Ty _Common_lerp(const _Ty _ArgA, const _Ty _ArgB, const _T
             }
         }
 
+        auto _Is_Overflow = [](_Ty val) constexpr {
+#ifdef _M_FP_STRICT
+            if (_STD is_constant_evaluated()) {
+                return _Is_inf(val);
+            }
+            return fetestexcept(FE_OVERFLOW) != 0;
+#else // ^^^ defined(_M_FP_STRICT) / !defined(_M_FP_STRICT) vvv
+            return _Is_inf(val);
+#endif // _M_FP_STRICT
+        };
+
+        auto _Clear_FE_OVERFLOW_flag = []() constexpr {
+#ifdef _M_FP_STRICT
+            if (!_STD is_constant_evaluated()) {
+                feclearexcept(FE_OVERFLOW);

```

Comment:
This clears `FE_OVERFLOW` if it was set before calling `lerp`. Unlike `errno`, IEEE 754 exception flags are sticky, and library functions should not clear any flags that were set before calling them.

There are ways to hide unwanted internal floating-point exceptions (see e.g. [`feholdexcept`](https://en.cppreference.com/w/cpp/numeric/fenv/feholdexcept)), but I would recommend against manipulating floating-point exceptions if you are not familiar with them or numerical algorithms in general.